Source/Krypton Components/Krypton.Ribbon/View Draw/ViewDrawRibbonTab.cs

```diff
@@ -153,9 +153,9 @@ protected override void Dispose(bool disposing)
                 if (_mementos != null!)
                 {
                     // Dispose of all the mementos in the array
-                    foreach (IDisposable? memento in _mementos)
+                    foreach (IDisposable memento in _mementos)
```

Comment:
This was done because when running the ribbon demos, the `_mementos` where null!

Reply:
Used a previous version, will reintroduce

Reply:
Reintroduced markers